bundle/manifests/backstage-operator.clusterserviceversion.yaml

```
@@ -301,6 +301,7 @@ spec:
   - email: jianrzha@redhat.com
     name: Jianrong Zhang
   maturity: alpha
+  minKubeVersion: 1.26.0
```

> Set minKuberVersion to 1.26.0 so that the operator can only be deployed on OpenShift 4.13+.

Why limit to 4.13+? I can see [here](https://access.redhat.com/support/policy/updates/openshift) that OCP 4.12 (and 4.11) are still in maintenance support, and the operator works fine on OCP 4.12 (I just tested it).

So I'd suggest setting it to 1.24.0 (which maps to OCP 4.11) instead (as done actually in the RHDH CSV: https://github.com/janus-idp/operator/blob/main/.rhdh/bundle/manifests/rhdh-operator.csv.yaml#L321), no?

```suggestion
  minKubeVersion: 1.24.0
```

---

As discussed, it would make sense to set the minimum version to OCP 4.12